.github/workflows/automation.yml

```diff
@@ -0,0 +1,12 @@
+name: Automation
```

Comment:
suggestion: What would you think about calling it something like Test Automation, or in some way getting "test" or "Cypress" or something into the name? I'm wondering if that might help pick it out a glance on the Actions page.